### PR TITLE
Bug fix for Jenkins summary log

### PR DESCRIPTION
**Pull request by crlishka**
Fix a bug caused by moving the function which writes the Jenkins summary log to lib_validation_testing.py

### CODE REVIEW

File: test/ci/lib_validation_testing.py [hunk 1]
```diff
@@ -456,7 +457,7 @@ def write_jenkins_resnet20_cifar10_description(refResults, ngResults,
         refAccuracy = float(refResults['accuracy'])
         ngAccuracy = float(ngResults['accuracy'])
 
-        acceptableDelta = refAccuracy * kAcceptableAccuracyDelta
+        acceptableDelta = refAccuracy * acceptableDelta
         deltaAccuracy = abs(refAccuracy - ngAccuracy)
 
         fOut.write( 'Resnet-CIFAR10 accuracy - ref: %5.4f, ngraph: %5.4f, delta %5.4f; ngraph %4.2fx slower; %d epochs'
```
Comment (sharathns93) [inline test/ci/lib_validation_testing.py]: carrying over my comment from the previous PR, can we change this to show 'ngraph is x times faster' instead?

Comment (crlishka) [reply test/ci/lib_validation_testing.py]: Good point.  I can make that change pretty quickly.  I'll push another commit and then you can re-review.

**Comment by crlishka** [comment]
Testing failed due to job timeout -- see NGRAPH-1729 for details.  I will rerun with TESTNOW at a time with fewer jobs in the pool, as a workaround.

**Comment by crlishka** [comment]
TESTNOW

**Pull request merged**